### ISSUE TITLE
How to use tensorflow from the source? I'm a beginner

### ISSUE THREAD

**Issue by brlouco** [infinitered/nsfwjs#466]
Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX AVX2
- This is the message that appears in the log file

**Comment by GantMan** [comment]
no worries!  So this is for your node server?

Are you using tfjs-node?  or tfjs-node-gpu?

**Comment by brlouco** [comment]
yes, i'm using tfjs-node. I made a whatsapp bot that detects these images
